wagtail/core/models.py

```diff
@@ -4754,7 +4740,7 @@ def __str__(self):
 
 
 class CommentReply(models.Model):
-    comment = models.ForeignKey(Comment, on_delete=models.CASCADE, related_name='replies')
+    comment = ParentalKey(Comment, on_delete=models.CASCADE, related_name='replies')
```

Comment:
Django model cluster doesn't actually fully support tested parental keys. I think one place this might be an issue is with page copying, it'll copy the comments but not the replies. Could you check that?

Reply:
Yep, as we discussed, added a ticket in to delete all comments on copy in the future.